vendor/github.com/loft-sh/utils/pkg/helm/values/k8s.go

```diff
@@ -8,35 +8,35 @@ import (
 )
 
 var K8SAPIVersionMap = map[string]string{
-	"1.24": "k8s.gcr.io/kube-apiserver:v1.24.3",
-	"1.23": "k8s.gcr.io/kube-apiserver:v1.23.9",
-	"1.22": "k8s.gcr.io/kube-apiserver:v1.22.12",
-	"1.21": "k8s.gcr.io/kube-apiserver:v1.21.14",
-	"1.20": "k8s.gcr.io/kube-apiserver:v1.20.15",
+	"1.24": "registry.k8s.io/kube-apiserver:v1.24.3",
```

Comment:
Since this is a different package, we need to change that in https://github.com/loft-sh/utils

Reply:
Hey @FabianKramm I've raised a matching pull request for the utils repo to address the registry deprecation there as well and keep both projects in sync: https://github.com/loft-sh/utils/pull/5

Let me know if that wasn't what you meant 🙏🏻

Reply:
https://github.com/loft-sh/utils/pull/5 Has been merged so will mark this as resolved.